microdnf/commands/remove/remove.cpp

```diff
@@ -76,8 +76,7 @@ RemoveCommand::RemoveCommand(Command & parent, const std::string & name) : Comma
 void RemoveCommand::run() {
     auto & ctx = static_cast<Context &>(get_session());
 
-    // Load system repositories
-    ctx.load_repos(true, false);
+    ctx.base.get_repo_sack()->get_system_repo()->load();
```

Comment:
The original code was `ctx.load_repos (true, false);`. This loads the system repository and set excludes/includes.
The system repo is now loaded directly and excludes/includes are not set.

Test:
I have the "iftop" package installed on my system.
With original code:
```
# microdnf5 --setopt=excludepkgs=iftop remove iftop
Updating repositories metadata and load them:
Waiting until sack is filled ...
Sack is filled.
Argument 'iftop' matches only excluded packages.
Nothing to do.
```
With the new code:
```
# microdnf5 --setopt=excludepkgs=iftop remove iftop
Package Arch Version Repository Size
Removing:
  iftop x86_64 1.0-0.26.pre4.fc35 <TODO: from_repo> 97.0 KiB

Transaction Summary:
  Removing: 1 packages

Is this ok [y / N]:
```

Reply:
Oh, I've missed this, I'll add an explicit call to `ctx.base.get_rpm_package_sack()->setup_excludes_includes();` the same way I have done it in the repoquery command, ok? I see it as a somewhat temporary solution until we figure out how to do this in a better way.